### PR TITLE
Updated json for CRR

### PR DESCRIPTION
**Pull request by v-ashwinik**
Updated the json to enable CRR

### CODE REVIEW

**Comment by v-ashwinik** [comment]
#sign-off

**Comment by Jim-Parker** [comment]
@nivnar , can you take a look and let us know if this is good to merge?

File: .openpublishing.publish.config.json [hunk 1]
```diff
@@ -33,6 +33,11 @@
             "path_to_root": "_themes.pdf",
             "url": "https://github.com/Microsoft/templates.docs.msft.pdf",
             "branch": "master"
+        },
+        {  
+            "path_to_root": "Contribute_include",
```
Comment (nivnar) [inline .openpublishing.publish.config.json:38]: Better to place CRR under the docset folder `contribute/` like below
` "path_to_root": "contribute/Contribute_include",`

Also `Contribute_include` value is like an identifier or token for docs-help-pr repo that will be used in the path for includes. Recommend a shorter one may be `cg` or `guide`?
example : ` "path_to_root": "contribute/cg",`

@Jim-Parker to confirm the identifier term to use on crr; whether `cg` or `guide` or `Contribute_include`


path_to_root | A relative folder path to repository root.
-- | --

Comment (Jim-Parker) [reply .openpublishing.publish.config.json:38]: Let's use "guide". This may cover more than includes as we figure out the sharing model. Thanks

**Review by nivnar — CHANGES_REQUESTED** [review]
Please update the config as per our conversation offline. Thanks

File: .openpublishing.publish.config.json (hunk 1 above)
Comment (nivnar) [inline .openpublishing.publish.config.json:38]:
```suggestion
            "path_to_root": "contribute/guide",
```

**Comment by v-ashwinik** [comment]
#sign-off

**Review by Jim-Parker — APPROVED** [review]

**Pull request merged**